content/cpp/concepts/math-functions/terms/isnan/isnan.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'isnan()'
+Description: 'returns non-zero value (true) if value in parameter is a NaN value; and zero (false) otherwise.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`isnan()`**  returns non-zero value (true) if value in parameter is a `NaN` value; and zero (false) otherwise.
+
+## Syntax
+
+```pseudo
+bool isnan(float x);  
+bool isnan(double x);  
+bool isnan(long double x);  
+bool isnan(integral x);
+```
+It returns 1 if x is `NaN`, otherwise it returns 0.
```

Comment:
```suggestion

The `x` parameter is a floating-point value that can be of type `float`, `double`, `long double`, or `int`. The return value is 1 if `x` is `NaN`. Otherwise, 0 is returned.
```